vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -16,12 +16,28 @@
 #
 # Inputs: $evm.root['service_template_provision_task'].dialog_options
 #
+# array_entry example - Classification::1
+def vmdb_object_from_array_entry(entry)
+  model, id = entry.split("::")
+  $evm.vmdb(model.to_s.downcase, id.to_i) if model && id
```

Comment:
There's no need for the .to_s, I don't think...and also, I don't think you need the downcase (might be wrong on the second point, but if i recall, .vmdb handle both cases)